pkg/csi/service/wcpguest/controller.go

```diff
@@ -1155,3 +1155,11 @@ func (c *controller) ListSnapshots(ctx context.Context, req *csi.ListSnapshotsRe
 	log.Infof("ListSnapshots: called with args %+v", *req)
 	return nil, status.Error(codes.Unimplemented, "")
 }
+
+func (c *controller) ControllerGetVolume(ctx context.Context, req *csi.ControllerGetVolumeRequest) (
+	*csi.ControllerGetVolumeResponse, error) {
+	ctx = logger.NewContextWithLogger(ctx)
```

Comment:
we do not need to initialize context and logger for Unimplemented interfaces.
we just need to return `return nil, status.Error(codes.Unimplemented, "")`